core/src/main/resources/db/cgds.sql

```diff
@@ -727,3 +729,10 @@ CREATE TABLE `info` (
 );
 -- THIS MUST BE KEPT IN SYNC WITH db.version PROPERTY IN pom.xml
 INSERT INTO info VALUES ('1.1.0');
+
+--1.2.0 specific changes:
```

Comment:
I think cgds.sql should be just a snapshot, otherwise, it becomes another migration.sql over time. Let's keep it clean and keep everything in create table statement.